server/src/test/java/io/pravega/schemaregistry/storage/impl/group/GroupTest.java

```diff
@@ -525,9 +525,13 @@ public void testDeleteSchema() {
         inMemoryGroup.addSchema(schemaInfo, HashUtil.getFingerprint(schemaInfo.getSchemaData().array()), groupProperties, eTag).join();
         eTag = inMemoryGroup.getCurrentEtag().join();
         schemaData = new byte[5];
+        schemaInfo = new SchemaInfo(anygroup, SerializationFormat.Custom, ByteBuffer.wrap(schemaData),
+                ImmutableMap.of());
+        inMemoryGroup.addSchema(schemaInfo, HashUtil.getFingerprint(schemaInfo.getSchemaData().array()), groupProperties, eTag).join();
+        eTag = inMemoryGroup.getCurrentEtag().join();
```

Comment:
i donot see you adding the lines to "delete" the schema and verify.. which is the crux of the problem here.